### DISCUSSION TITLE
Find fork with youngest / latest commit

### DISCUSSION THREAD

**Discussion by buhtz** [community/community#40196] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello Folks,

I often have this situation. I find a repo which seems unmaintained based on the last commit date. But there are a lot of forks. And now I want to find out which one of that forks is the freshest. Maybe this is a fork which take over the project.

Example https://github.com/python-openxml/python-docx/network/members

Here we have over 900 forks.

Is there a way to check that automatically?

**Comment by esthet1cs** [comment]
I had the exact same question and found this tool which provides a table with all forks and the possibility to sort the table according to the date of the last push, size etc.
Git Repo: https://github.com/techgaun/active-forks
Live site: https://techgaun.github.io/active-forks/index.html
Use as bookmarklet: https://github.com/techgaun/active-forks#as-bookmarklet
